saleor/graphql/meta/utils.py

```diff
@@ -0,0 +1,13 @@
+from ...checkout.models import Checkout
```

Comment:
I have two issues with this:
1. This seems to belong to checkout as it's only operating on the checkout piece. Would it make sense to move it to `graphql.checkout`?
2. We shouldn't be creating *new* `utils.py` files as they become buckets containing a lot of random stuff and instead create something more meaningful. `checkout.utils` is actually a good example of that because it has logic manipulating checkout in place, creating new objects, operating on discounts... all of it 😅 I'd suggest adding it to checkout `utils.py` for now and then splitting this file into specific modules, each responsible for a specific set of functions.

Reply:
I missed the `graphql/meta/mutation/utils.py` file, so adding a new one was unnecessary - fixed. I agree that this is not a perfect pattern, as indeed utils files are already buckets of different stuff. We will have to refactor it at some point.